webapp/src/test/java/io/github/microcks/web/RestControllerIT.java

```diff
@@ -88,4 +91,32 @@ public void testSwaggerMocking() {
          fail("No Exception should be thrown here");
       }
    }
+   
+   @Test
+   public void testFallbackMatchingWithRegex() {
+      // Upload modified pastry spec
+      uploadArtifactFile("target/test-classes/io/github/microcks/util/openapi/pastry-with-details-openapi.yaml", true);
+
+      // Check operation with a defined mock (name: 'Millefeuille')
+      ResponseEntity<String> response = restTemplate.getForEntity("/rest/pastry-details/1.0.0/pastry/Millefeuille/details", String.class);
+      assertEquals(200, response.getStatusCode().value());
+      try {
+         JSONObject details = new JSONObject(response.getBody());
+         String description = details.getString("description");
+         assertTrue(description.startsWith("Detail -"));   
+      } catch (Exception e) {
+    	 fail("No Exception should be thrown here");
```

Comment:
I don't know whats happening with these indents - copy paste just does not work on my site
Do you have a code formatter rule or anything I can use to apply?